src/codec.ts

```diff
@@ -176,6 +176,10 @@ function generateToJSONFromRow(row) {
         }
       }
 
+      if (value instanceof Date) {
+        value = value.toString();
```

Comment:
I think we'll probably want to call `toJSON()` instead of `toString()` here. @stephenplusplus have any opinions here?

Reply:
(new Date()).toJSON() => '2018-11-20T16:08:06.026Z'
(new Date()).toString() => 'Tue Nov 20 2018 17:08:12 GMT+0100 (Central European Standard Time)'